content.md

```diff
@@ -88,6 +87,13 @@ This list is also available online at: https://resources.osucord.moe/
 - [Ronnia](https://ronnia.me/) - Twitch/osu! bot that sends beatmap requests from Twitch chat to the streamer's in-game messages
 - [JKPS](https://github.com/Tonetfal/JKPS) - Keyboard overlay featuring key press visualisation and a keys per second meter
 ---
+### Discord Bots
+- [owo](http://owo-bot.xyz/) - A general-purpose discord bot to show of your profile/recent plays, get map recommendations and compare yourself to others
+- [BathBot](https://github.com/MaxOhn/Bathbot) - A feature-rich discord bot with functionality all around osu!
```

Comment:
nowadays bathbot is a lot better, more feature-rich and kept more up-to-date, so I think it should be above owo, should also have a description closer to what owo has

Reply:
Right now the resources in general in each category aren't exactly ordered, but considering reading order, yes.

Reply:
yeah because I haven't bothered doing an in-depth look over the whole document yet lol